### PR TITLE
added advertisment flag for hero module

### PR DESCRIPTION
**Pull request by MarkoDeveloper1407**
# Description

Added a flag which checks if the hero is an advertisment banner. If so renders image in a different place to be inline with the natural way HTML is rendered.

Related to: https://github.com/uswitch/contentful-components/pull/509

image
image

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [x] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [x] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [x] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [x] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/elements/hero/src/index.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
   className?: string
   variant?: 'provider'
   bottomImageOverflow?: number | string
+  advertisment?: boolean
```
Comment (erkinun) [inline src/elements/hero/src/index.tsx:33]: same typo here

File: src/elements/hero/src/index.tsx [hunk 2]
```diff
@@ -83,47 +85,50 @@ const Hero: React.FC<Props> = ({
...
-                  display: [
-                    fgImageOnMobile ? 'block' : 'none',
-                    fgImageOnTablet ? 'block' : 'none',
-                    'block'
-                  ],
-                  ...(fgImage && fgImageType === 'background'
-                    ? {
-                        backgroundImage: `url(${fgImage})`,
-                        backgroundRepeat: 'no-repeat',
+              {!advertisment ? (
```
Comment (erkinun) [inline src/elements/hero/src/index.tsx:96]: Can you use && for better readability? The `null` after the `:` is unnecessary.

**Review by erkinun — CHANGES_REQUESTED** [review]
looks good, just a type and a nitpick comment

File: src/elements/hero/src/index.tsx (hunk 1 above)
Comment (MarkoDeveloper1407) [reply src/elements/hero/src/index.tsx]: Done

File: src/elements/hero/src/index.tsx (hunk 2 above)
Comment (MarkoDeveloper1407) [reply src/elements/hero/src/index.tsx]: Done

**Comment by erkinun** [comment]
we'll implement this in a better way, thus closed the PR

**Pull request closed without merging**